OrbitCore/CaptureData.h

```diff
@@ -132,6 +140,21 @@ class CaptureData {
     callstack_data_->AddCallstackEvent(std::move(callstack_event));
   }
 
+  void AddUniqueTracepointEventInfo(uint64_t key,
+                                    orbit_grpc_protos::TracepointInfo tracepoint_info) {
+    tracepoint_info_manager_->AddUniqueTracepointEventInfo(key, std::move(tracepoint_info));
+  }
+
+  void AddTracepointEvent(orbit_client_protos::TracepointEventInfo tracepoint_event_info) {
+    tracepoint_info_manager_->AddTracepointEvent(std::move(tracepoint_event_info));
+  }
+
+  void AddTracepointEvent(uint64_t time, uint64_t tracepoint_hash, int32_t thread_id){
+    tracepoint_event_buffer_->AddTracepointEvent(time,
+                                                 tracepoint_hash,
+                                                 thread_id);
+  }
```

Comment:
Do/will you use/need both these methods or is only one of the two used? In the latter case, I would remove the one that is not necessary, let's try to have only necessary methods in this big class.

Reply:
I use both of them.